README.md

```diff
@@ -37,6 +37,13 @@ For the prototype, source data will live in the folder `data/`. Really large fil
 To change settings specific to an individual machine (such as the data directory), edit the contents of `.env` 
 in the root directory of your project. A `.env-sample` file is provided as a template.
 
+For University of Michigan internal development, files can be found at the following locations:
+
+Tutorial videos: /net/amd/amkwong/media/
```

Comment:
Future improvement note: Is there any reason that the FIVEx media files are not in the FIVEx folder? (taken by itself, `media` is a really generic name)

This relates to the "scavenger hunt" aspect of how we handle large assets. I could imagine a hierarchy that kept everything together: `amkwong/fivex/data`, `amkwong/fivex/media`... etc.

That said, I've confirmed that I can access the project folders as a non-amd member and that the files have read permissions for me. Thanks for gathering assets together!